helpers/bench.nix

```diff
@@ -0,0 +1,93 @@
+{ snabbSrc ? (builtins.fetchTarball https://github.com/snabbco/snabb/tarball/next)
+# specify how many times is each benchmark ran
+, numTimesRunBenchmark ? 20
+# specify on what hardware will the benchmarks be ran
+, requiredSystemFeatures ? [ "performance" ]
+, SNABB_PCI0 ? "0000:01:00.0"
+, SNABB_PCI_INTEL0 ? "0000:01:00.0"
+, SNABB_PCI_INTEL1 ? "0000:01:00.1"
```

Comment:
I am worried about these PCI device IDs being used on the wrong machine and potentially causing problems like knocking servers off the internet by taking over the wrong network card.

Could we move these default values into a machine-specific config file like [lab.nix](https://github.com/snabblab/snabblab-nixos/blob/master/machines/lab.nix) instead? Perhaps as global environment variables?